go.mod

```diff
@@ -1,11 +1,15 @@
-module github.com/BuxOrg/bux-server
+module github.com/bitcoin-sv/spv-wallet
 
 go 1.21.5
 
+replace github.com/bitcoin-sv/spv-wallet/models => ./models
+
+replace github.com/bitcoin-sv/spv-wallet/engine => ./engine
```

Comment:
For the sake of being clear and to prevent removing those replaces, maybe we should put here the comments that we're using those packages directly to make easier building a cohesive features that often would need to touch all of those packages.